examples/src/main/java/org/apache/ignite/examples/client/ClientKubernetesPutGetExample.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.examples.client;
+
+import org.apache.ignite.Ignition;
+import org.apache.ignite.client.ClientCache;
+import org.apache.ignite.client.IgniteClient;
+import org.apache.ignite.client.ThinClientKubernetesAddressFinder;
+import org.apache.ignite.configuration.ClientConfiguration;
+import org.apache.ignite.examples.model.Address;
+import org.apache.ignite.kubernetes.configuration.KubernetesConnectionConfiguration;
+
+/**
+ * Demonstrates how to use Ignite thin client within the Kubernetes cluster using KubernetesConnectionConfiguration.
+ * <p>
+ * Prerequisites:
+ * <ul>
+ * <li>Running Ignite Kubernetes cluster. Check modules/kubernetes/DEVNOTES.md for example how to run a cluster locally.</li>
+ * <li>A thin client application should be run within the cluster to have access to Ignite nodes pods.</li>
```

Comment:
```suggestion
 * <li>A thin client application should be run within the cluster to provide access to the Ignite nodes pods.</li>
```

Reply:
Thin client does not "provide" access to Ignite, but do "have" the access to it within the cluster. I mean that there is no access to cluster if thin client isn't running within the cluster.

Reply:
OK, got it, thanks for the explanation!